jupyterhub_singleuser_profiles/profiles.py

```diff
@@ -179,18 +180,20 @@ def get_profile_by_name(self, name):
   def setup_services(self, spawner, image, user):
     profile = self.get_merged_profile(image, user)
     if profile.get("services"):
-      for key, service in profile.get("services").items():
-        template = self.service.get_template(service["template"])
-        if not template:
-          _LOGGER.warning("Could not load the template %s. Skipping setting up the service %s" % (service["template"], key))
-          continue
-        resource = self.service.process_template(user, template, **service.get("parameters", {}))
-        envs = self.service.submit_resource(resource, service.get("return", {}))
-        spawner.environment = {**spawner.environment, **envs}
-        spawner.single_user_services.append(resource.get("metadata").get("name"))
+      for service_name, service in profile.get("services").items():
```

Comment:
I think we should move this `for` loop into a method in `services.py` to keep a good separation